app/views/content_items/_body_with_related_links.html.erb

```diff
@@ -0,0 +1,20 @@
+<% content_for :simple_header, true %>
+<%= content_for :title, @content_item.title %>
+
+<div class="grid-row sidebar-with-body">
+  <div class="column-two-thirds">
+    <%= render 'govuk_component/title',
+        title: @content_item.title %>
+    <%= render 'govuk_component/govspeak',
+        content: @content_item.body,
+        direction: page_text_direction %>
+    <div class="meta-data group">
```

Comment:
Why do we need the `group` class?

The `<p class="modified-date">` container should be enough. We can remove the `div` container completely.